page-full-no.php

```diff
@@ -1,22 +1,27 @@
 <?php
-
-/* Template Name: MOH Full Width */
-
+/**
+ * Template Name: MOH Full Width
```

Comment:
This `MOH Full Width` name is currently shared between two templates (page-full-no.php and page-moh-about.php), which may make one not possible to select.